nearbridge/contracts/NearBridge.sol

```diff
@@ -75,10 +81,11 @@ contract NearBridge is INearBridge {
         bytes32 blockHash
     );
 
-    constructor(Ed25519 ed, uint256 _lockEthAmount, uint256 _lockDuration) public {
+    constructor(Ed25519 ed, uint256 _lockEthAmount, uint256 _lockDuration, uint256 _replaceDuration) public {
```

Comment:
I really wish the code to deploy the contracts were in the same repository as the contracts themselves. Currently, I cannot merge this PR because it fails CI.

Reply:
I agree. CC @Kouprin